tests/phpunit/tests/multisite/wpMsUsersListTable.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+if ( is_multisite() ) :
```

Comment:
`@group ms-required` should prevent this from running in single site mode. 


:1234: Applies elsewhere too.

Reply:
Good point! Fixed in [d26fdc0](https://github.com/WordPress/wordpress-develop/pull/2828/commits/d26fdc07e5bcb0311b69b463269f73cb56bb58e8).